### ISSUE TITLE
Unable to achieve the same high mAP value as in the paper

### ISSUE THREAD

**Issue by humanoid-konjac** [924973292/IDEA#7]
Thank you very much for your outstanding work!

However, I encountered some issues when attempting to reproduce your paper. I directly cloned the code from GitHub and configured the environment strictly according to the guide, using the pre-trained model and the RGBNT201 dataset you provided in README. After reviewing the training log file you uploaded and found that our configurations are identical except for the pre-trained model path. However, I still cannot achieve an mAP above 80%.

In other issues, I noticed you mentioned the problem might be caused by shared cluster usage. However, I tested both the RTX 3090 and RTX 4090, and no one else was using these graphics cards during training, yet there was still no improvement.

I want to conduct research on pedestrian re-identification, so vehicle datasets like RGBNT100 are not particularly helpful for my purposes.

Below are the final results of my training. My sincere thanks for your assistance.

<img width="1350" height="863" alt="Image" src="https://github.com/user-attachments/assets/947f3f71-3819-4cd6-872d-64ae3e4f7e1e" />

LOG File: [train_log.txt](https://github.com/user-attachments/files/22379970/train_log.txt)

**Comment by 924973292** [comment]
Thank you for your attention. I have already shared my reproduction results under another issue. If your machine does not meet the requirements, please update the entire set of hyperparameters and optimizer configurations (I recently found that stability improves with some new attempts). Make sure to align with the two-stage configuration of CLIP-ReID, and try setting the batch size and number of instances to 64 and 8 respectively to see how the results turn out.

**Comment by liyali337** [comment]
> Thank you very much for your outstanding work!
> 
> However, I encountered some issues when attempting to reproduce your paper. I directly cloned the code from GitHub and configured the environment strictly according to the guide, using the pre-trained model and the RGBNT201 dataset you provided in README. After reviewing the training log file you uploaded and found that our configurations are identical except for the pre-trained model path. However, I still cannot achieve an mAP above 80%.
> 
> In other issues, I noticed you mentioned the problem might be caused by shared cluster usage. However, I tested both the RTX 3090 and RTX 4090, and no one else was using these graphics cards during training, yet there was still no improvement.
> 
> I want to conduct research on pedestrian re-identification, so vehicle datasets like RGBNT100 are not particularly helpful for my purposes.
> 
> Below are the final results of my training. My sincere thanks for your assistance.
> 
> <img alt="Image" width="1350" height="863" src="https://private-user-images.githubusercontent.com/112099388/490436229-947f3f71-3819-4cd6-872d-64ae3e4f7e1e.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTg3MDQyMzIsIm5iZiI6MTc1ODcwMzkzMiwicGF0aCI6Ii8xMTIwOTkzODgvNDkwNDM2MjI5LTk0N2YzZjcxLTM4MTktNGNkNi04NzJkLTY0YWUzZTRmN2UxZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwOTI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDkyNFQwODUyMTJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iOTQ1MmI0NmQzNjM1YzJiMmZiNmFjNWM4ZDc1OTI4MDI2ZmZmMjBiMzRiZTAyY2JmOGRkMGM2ODQxODI3Nzk2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.p0RTWQ_lrA0LDqIdO1IavIpMf4LKpdHbeJAyRiTHBZ4">
> LOG File: [train_log.txt](https://github.com/user-attachments/files/22379970/train_log.txt)

Hello, may I ask if you are able to obtain the same results when repeating the experiment under the same settings?

**Comment by humanoid-konjac** [comment]
> > Thank you very much for your outstanding work!
> > However, I encountered some issues when attempting to reproduce your paper. I directly cloned the code from GitHub and configured the environment strictly according to the guide, using the pre-trained model and the RGBNT201 dataset you provided in README. After reviewing the training log file you uploaded and found that our configurations are identical except for the pre-trained model path. However, I still cannot achieve an mAP above 80%.
> > In other issues, I noticed you mentioned the problem might be caused by shared cluster usage. However, I tested both the RTX 3090 and RTX 4090, and no one else was using these graphics cards during training, yet there was still no improvement.
> > I want to conduct research on pedestrian re-identification, so vehicle datasets like RGBNT100 are not particularly helpful for my purposes.
> > Below are the final results of my training. My sincere thanks for your assistance.
> > <img alt="Image" width="1350" height="863" src="https://private-user-images.githubusercontent.com/112099388/490436229-947f3f71-3819-4cd6-872d-64ae3e4f7e1e.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTg3MDQyMzIsIm5iZiI6MTc1ODcwMzkzMiwicGF0aCI6Ii8xMTIwOTkzODgvNDkwNDM2MjI5LTk0N2YzZjcxLTM4MTktNGNkNi04NzJkLTY0YWUzZTRmN2UxZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwOTI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDkyNFQwODUyMTJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iOTQ1MmI0NmQzNjM1YzJiMmZiNmFjNWM4ZDc1OTI4MDI2ZmZmMjBiMzRiZTAyY2JmOGRkMGM2ODQxODI3Nzk2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.p0RTWQ_lrA0LDqIdO1IavIpMf4LKpdHbeJAyRiTHBZ4">
> > LOG File: [train_log.txt](https://github.com/user-attachments/files/22379970/train_log.txt)
> 
> Hello, may I ask if you are able to obtain the same results when repeating the experiment under the same settings?

No. And I don't know how to update the hyperparameters and optimizer configuration. I tried “aligning with the two-stage configuration of CLIP-ReID,” but IDEA only has a single training stage, so I'm not quite sure how to align it with CLIP-ReID's configuration.😭

**Comment by 924973292** [comment]
@humanoid-konjac Use the second-stage's params instead of “two-stage”.👍

**Comment by humanoid-konjac** [comment]
@924973292 Thank you, but I'd like to confirm if you're referring to this set of parameters? Because after training and testing, when I changed the parameters in IDEA.yml to this set, it seems that due to an excessively low learning rate, the final mAP on RGBNT201 only reached just over 20%.

<img width="1224" height="1138" alt="Image" src="https://github.com/user-attachments/assets/de3c711c-7d40-4e86-b772-fcf61698585d" />

If this isn't the set of hyperparameters you mentioned, could you please provide the more stable hyperparameters you referred to? I greatly appreciate your guidance.
